### PR TITLE
Chore: add footer credit details in the html

### PR DESCRIPTION
**Pull request by JonasSkjodt**
Adds credits to the footer [#111](https://github.com/NovoNordisk-OpenSource/decentralized-tech-radar/issues/111)

### CODE REVIEW

**Review by Slug-Boi — APPROVED** [review]

File: src/HTML/makeHtml.go [hunk 1]
```diff
@@ -39,13 +39,12 @@ func GenerateHtml(csvData string) {
 			</p>
 			</div>
 			<div class="all-quadrants-mobile show-all-quadrants-mobile"></div>
-			<div class="graph-footer"></div>
+			<div class="graph-footer">
+				<p class="agree-terms">Visit the Novo Nordisk <a href="https://github.com/NovoNordisk-OpenSource/decentralized-tech-radar">Tech Radar open source on Github</a>. This tech radar was heavily inspired by Thoughtworks.</p>
```
Comment (Haruyzal) [inline src/HTML/makeHtml.go:43]: Is it okay to not link to Thoughtworks?

Comment (JonasSkjodt) [reply src/HTML/makeHtml.go:43]: It's ok yes, but I think you're right - we should show more appreciation and link directly to them. That being said, the message might change after Lasse has looked at it, so we can change it at that time.

**Review by August-Brandt — APPROVED** [review]
Looks good

File: src/HTML/makeHtml.go (hunk 1 above)
Comment (Slug-Boi) [reply src/HTML/makeHtml.go:43]: I will write a message @ him on discord and mark this as resolved

**Pull request merged**